content/sites/start-from-scratch.md

```diff
@@ -33,6 +33,7 @@ To start from scratch to create a Workers Site, follow these steps:
      zone_id = "42ef.."
      route = "example.com/*"
      ```
+   (Note: the wildcard selector is important to ensure that paths to your assets match and are loaded properly) 
```

Comment:
```suggestion
   (Note: the wildcard selector is important to ensure that paths to your assets match, and are loaded properly) 
```

Reply:
Just a quick little suggestion here, once that's added, happy to merge! Thank you :)

Reply:
Awesome!